packages/app/src/systems/Account/services/account.ts

```diff
@@ -1,10 +1,9 @@
 /* eslint-disable consistent-return */
 import type { WalletUnlocked } from '@fuel-ts/wallet';
-import { WalletManager } from '@fuel-ts/wallet-manager';
 import type { Account } from '@fuel-wallet/types';
 import { bn, Address, Provider } from 'fuels';
 
-import { IndexedDBStorage } from '../utils';
+import { unlockManager } from '../utils/manager';
```

Comment:
add manager to `index.tsx` inside utils file, then fix import to use `utils` directly, instead of `utils/manager`